api/src/main/java/edu/cornell/mannlib/vitro/webapp/auth/checks/SparqlSelectQueryResultsChecker.java

```diff
@@ -26,15 +27,19 @@
 import org.apache.jena.rdf.model.RDFNode;
 
 public class SparqlSelectQueryResultsChecker {
+    private static final String VAR_PROFILE_URI = "profileUri";
+    private static final String VAR_EXTERNAL_AUTH_ID = "externalAuthId";
+    private static final String VAR_MATCHING_PROPERTY_URI = "matchingPropertyUri";
+    private static final String VAR_OBJECT_URI = "objectUri";
```

Comment:
Can I ask why you decided to add a prefix VAR? Did we use this practice in the past in VIVO?

Reply:
Because that's all sparql variables, I removed VAR_ prefix.

Reply:
thanks